src/main/xerus/monstercat/MonsterUtilities.kt

```diff
@@ -102,20 +102,20 @@ fun showErrorSafe(error: Throwable, title: String = "Error") {
 }
 
 class MonsterUtilities : VBox(), JFXMessageDisplay {
-	
+
```

Comment:
ändere deinen codestyle für das Projekt bitte so, dass sowas sich nicht verändert `Tabs and Indents > Keep indents on empty lines`. Ich finde das angenehmer.

Reply:
Wird noch geändert